tests/cupyx_tests/scipy_tests/sparse_tests/test_linalg.py

```diff
@@ -169,18 +169,14 @@ def test_dense(self, dtype, xp, sp):
             a = sp.linalg.aslinearoperator(a)
         return self._test_eigsh(a, xp, sp)
 
-    @pytest.mark.xfail(
-        reason='eigsh works wrong (#5001)',
-        raises=AssertionError,
-    )
     @testing.for_dtypes('fdFD')
     @testing.numpy_cupy_allclose(rtol=tol, atol=tol, sp_name='sp')
     def test_dense_low_rank(self, dtype, xp, sp):
         n = self.n
         rank = 5
         # density is ignored.
-        a = testing.shaped_random((n, rank), xp, dtype=dtype, scale=1).dot(
-            testing.shaped_random((rank, n), xp, dtype=dtype, scale=1))
+        a = testing.shaped_random((n, rank), xp, dtype=dtype, scale=1)
+        a = a @ a.T
```

Comment:
Thanks for fixing it.  For complex dtypes, apply `.conj()`, too.

```suggestion
        a = a @ a.conj().T
```